python/python/bystro/ancestry/train_utils.py

```diff
@@ -0,0 +1,24 @@
+"""Utilities for ancestry model training."""
+
+from typing import Any
+
+import numpy as np
+
+
+def get_variant_ids_from_callset(callset: dict[str, Any]) -> np.ndarray:
```

Comment:
definitely not something to do in this PR, but I may think about making a custom Struct or attrs slotted class for `callset` instead of a dict.